checks/http/ucoz.rb

```diff
@@ -0,0 +1,31 @@
+module Intrigue
+  module Ident
+  module Check
+  class uCoz < Intrigue::Ident::Check::Base
```

Comment:
Same, more info here: http://ruby-for-beginners.rubymonstas.org/writing_classes/definition.html